disque/connection_test.go

```diff
@@ -422,10 +423,12 @@ func (s *DisqueSuite) TestFetchWithMultipleJobs() {
 	jobs, err := d.FetchMultiple("queue5", 2, time.Second)
 	assert.Nil(s.T(), err)
 	assert.Equal(s.T(), 2, len(jobs))
+	assert.Equal(s.T(), 2, d.count)
```

Comment:
Not sure about this one. You may have intended to increment by one on every FetchMultiple call instead of the count argument.